src/main/java/seedu/workbook/model/internship/DateTime.java

```diff
@@ -7,21 +7,22 @@
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeFormatterBuilder;
 import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
 
 /**
  * Represents a Date in WorkBook.
  * Guarantees: immutable; name is valid as declared in {@link #isValidDate(String)}
  */
 public class DateTime {
     public static final DateTime EMPTY_DATETIME = new DateTime("");
-    public static final String MESSAGE_CONSTRAINTS = "Date should be dd-mmm-yyyy hh:mm";
+    public static final String MESSAGE_CONSTRAINTS = "Date should be dd-mmm-yyyy hh:mm, and should be valid.";
```

Comment:
"and should be valid" sounds vague to me, perhaps could consider something like:
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Date should be formatted as dd-mmm-yyyy hh:mm, and should actually exist.";
```